druid-shell/src/window.rs

```diff
@@ -92,6 +92,17 @@ impl IdleToken {
     }
 }
 
+/// Levels in the window system - Z order for display purposes.
+/// Describes the purpose of a window and should be mapped appropriately to match platform
+/// conventions.
+#[derive(Copy, Clone, Debug)]
+pub enum WindowLevel {
+    AppWindow,
+    Tooltip,
+    DropDown, // Eg in a combo box or custom menu
```

Comment:
It would be slightly more friendly to have a short docstring in each variant, which would be a good place for this clarification.